test/jdk/sun/util/resources/TimeZone/Bug4848242.java

```diff
@@ -35,7 +35,6 @@
 public class Bug4848242 {
```

Comment:
This test needs a copyright year bump. (Applies to any other tests as well).

We also might want to update the summary, since it should no longer display MET and MEST, but rather CET and CEST.